api/client/build.go

```diff
@@ -209,6 +210,15 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 		}
 	}
 
+	var shmSize int64 = 67108864 // initial SHM size is 64MB
```

Comment:
Hi,
I added the description of the default value to the help text. 
Is it good in such a description?